File: ColegioTrenerAndroid/app/src/main/java/com/jjmf/colegiotrenerandroid/domain/model/Pago.kt
package com.jjmf.colegiotrenerandroid.domain.model

import java.util.Date

data class Pago(
    val anoaca: String?,
    val codconcepto: String?,
    val concepto: String?,
    val fecven: Date,
    val saldo: Double,
    val estado: String?,
    val tipdoc: String?,
    val numdoc: String?,
    val mediopago: String?,
    val fecpag: Date,
    val importepagado: Double,
    val mora: Double,
)

File: ColegioTrenerSwift/ColegioTrenerSwift/Domain/Model/Pago.swift
//
//  Pago.swift
//  ColegioTrenerSwift
//
//  Created by Joel Maldonado on 25/04/24.
//

import SwiftUI

struct Pago: Hashable {
    let anoaca: String
    let codconcepto: String
    let concepto: String
    let fecven: Date
    let saldo: Double
    let estado: String
    
    //Realizado
    let tipdoc: String
    let numdoc: String
    let mediopago: String
    let fecpag: Date
    let importepagado: Double
    let mora: Double
}
